src/constants/tokens.ts

```diff
@@ -0,0 +1,313 @@
+import { SupportedChainId } from 'packages/BalancedJs/chain';
+
+import { Token } from 'types/balanced-sdk-core/index';
+
+import { NETWORK_ID } from './config';
+
+export const ICX = new Token(SupportedChainId.MAINNET, 'cx0000000000000000000000000000000000000000', 18, 'ICX', 'ICX');
+export const sICX = new Token(
+  SupportedChainId.MAINNET,
+  'cx2609b924e33ef00b648a409245c7ea394c467824',
+  18,
+  'sICX',
+  'Staked ICX',
+);
+export const bnUSD = new Token(
+  SupportedChainId.MAINNET,
+  'cx88fd7df7ddff82f7cc735c871dc519838cb235bb',
+  18,
+  'bnUSD',
+  'Balanced Dollar',
+);
+export const BALN = new Token(
+  SupportedChainId.MAINNET,
+  'cxf61cd5a45dc9f91c15aa65831a30a90d59a09619',
+  18,
+  'BALN',
+  'Balance Token',
+);
+export const IUSDC = new Token(
+  SupportedChainId.MAINNET, //
+  'cxae3034235540b924dfcc1b45836c293dcc82bfb7',
+  6,
+  'IUSDC',
+  'ICON USD Coin',
+);
+export const USDS = new Token(
+  SupportedChainId.MAINNET, //
+  'cxbb2871f468a3008f80b08fdde5b8b951583acf06',
+  18,
+  'USDS',
+  'Stably USD',
+);
+export const OMM = new Token(
+  SupportedChainId.MAINNET, //
+  'cx1a29259a59f463a67bb2ef84398b30ca56b5830a',
+  18,
+  'OMM',
+  'Omm Token',
+);
+export const CFT = new Token(
+  SupportedChainId.MAINNET, //
+  'cx2e6d0fc0eca04965d06038c8406093337f085fcf',
+  18,
+  'CFT',
+  'Craft',
+);
+export const METX = new Token(
+  SupportedChainId.MAINNET, //
+  'cx369a5f4ce4f4648dfc96ba0c8229be0693b4eca2',
+  18,
+  'METX',
+  'Metanyx',
+);
+export const IUSDT = new Token(
+  SupportedChainId.MAINNET, //
+  'cx3a36ea1f6b9aa3d2dd9cb68e8987bcc3aabaaa88',
+  18,
```

Comment:
@octocat512 It seems IUSDT decimals should be 6

Reply:
Good Catch! Thanks